### ISSUE TITLE
Error Code: 10124

### ISSUE THREAD

**Issue by vxlm** [zoom/meetingsdk-win-raw-recording-sample#1]
Has anyone encountered this error trying to set this demo up? I have gone through the non-patched demo, and managed to get joining working. However, when I try to join the same meeting in this demo, I get `LastErrorType_Auth, Error Code 10124.`

According to the [forums ](https://devforum.zoom.us/t/demo-app-auth-failed-with-error-code-10124/27150/5)this is the most likely due to JWT token payload having the wrong format - and for the non-patched demo, I manually generated the JWT token. 

I'm not too familiar with how C# works, and I have been trying to find where the JWT payload is defined to debug it, but have not been successful in finding it.

If anyone has managed to get this demo working, please let me know! Thanks.

**Comment by ADityaraj3** [comment]
Did you solve this? If yes than can you please guide me.

**Comment by ADityaraj3** [comment]
I have solved the issue, anyone can ask me if they are stuck thanks.

**Comment by TOGUNOLUWATOBI** [comment]
@ADityaraj3 
I am currently having the same issue. Can you kindly guide me on how to go about it

**Comment by SardarkhanStella** [comment]
Yes this issue is about sdk key or SDK secret. that's why this issue occurs.

**Comment by analog88** [comment]
Hey @vxlm, if you're running into issues with JWT token errors like 10124 while setting up your meeting bot, you might want to check out the [Recall.ai API](https://www.recall.ai/?utm_source=github&utm_medium=forum_answer&utm_campaign=error_code_10124&utm_content=Recall.ai_API).

It’s a simple 3rd party API that lets you use meeting bots to get raw audio/video/metadata from meetings without you needing to spend months to build, scale and maintain these bots.

Here are the API docs: https://docs.recall.ai/page/tutorials
